src/pages/centers.tsx

```diff
@@ -401,23 +501,37 @@ const Center: React.FC = () => {
     }
   };
 
-  const handleEdit = (rowData: any) => {
+  const handleEdit = async (rowData: any) => {
     setLoading(true);
     // Handle edit action here
-    setIsEditModalOpen(true);
+    // setIsEditModalOpen(true);
     if (rowData) {
       const cohortId = rowData?.cohortId;
       setSelectedCohortId(cohortId);
       const cohortName = rowData?.name;
       setInputName(cohortName);
+
+      let data = {
+        filters: {
+          cohortId: cohortId,
+        },
+        limit: 200,
```

Comment:
@upendraTekdi why limit is 200, reduce it to 10 or 20

Reply:
done